tasks/Module 4/task_6.py

```diff
@@ -0,0 +1,20 @@
+"""Brute-Force PDF Password Breaker."""
+import os
+import PyPDF2
+
+
+def break_pdf_password(pdf_file, pass_dict_file):
+    """Brute-Force PDF Password Breaker."""
+    pdf_reader = PyPDF2.PdfFileReader(open(pdf_file, 'rb'))
```

Comment:
Since you are opening a file here it would be good to use context manager for it as well. Maybe:
```python
    with open(pass_dict_file, 'r') as pwd_file, open(pdf_file, 'rb') as pdf_file:
        pdf_reader = PyPDF2.PdfFileReader(pdf_file)
        for word in pwd_file:
            ...
```
I know that `pdf_file` is already a name of function argument, so you can rename one of those variables. I usually name variables with filenames like `something_filename`, and those with file objects like `something_file`.